content/sensu-enterprise/3.0/upgrading.md

```diff
@@ -0,0 +1,159 @@
+
+The OpsGenie v2 Alert API replaces the `teams` and `recipients` attributes with
+a new `responders` attribute. As a result you must upgrade your Sensu
+Enterprise configuration for OpsGenie in order to correctly route
+alerts using the new API.
+
+Example OpsGenie configuration for Sensu Enterprise prior to 3.0:
+
+{{< highlight json >}}
+{
+  "opsgenie": {
+    "api_key": "eed02a0d-85a4-427b-851a-18dd8fd80d93",
+    "teams": ["ops", "web"],
+    "recipients": ["afterhours"]
+  }
+}
+{{</ highlight >}}
+
+Assuming `afterhours` is an escalation plan, the values supplied for
+`teams` and `recipients` above can be translated to the new
+`responders` attribute like so:
+
+{{< highlight json >}}
+{
+  "opsgenie": {
+    "api_key": "eed02a0d-85a4-427b-851a-18dd8fd80d93",
+    "responders": [
+      {
+        "type": "team",
+        "name": "ops"
+      },
+      {
+        "type": "team",
+        "name": "web"
+      },
+      {
+        "type": "escalation",
+        "name": "afterhours"
+      }
+    ]
+  }
+}
+{{</ highlight >}}
+
+As shown above, the `responders` attribute expects an array of hashes
+specifying the `type` and `name` for each object. Depending on the
+given type of a responder, the identifying attribute (e.g. `name`) may
+vary. Please see [OpsGenie's Alert API Migration
+Guide][opsgenie-api-migration] for more details.
+
+#### Update `overwrites_quiet_hours`
+
+Sensu Enterprise 3.0 updates the name of the `overwrites_quiet_hours` attribute
+to `overwrite_quiet_hours`. The singular form of this attribute is required to
+achieve the desired result of overriding alert filtering that would otherwise
+prevent OpsGenie from notifying recipient(s) during their configured quiet hours.
+
+Example OpsGenie configuration for Sensu Enterprise 3.0:
+
+{{< highlight json >}}
+{
+  "opsgenie": {
+    "api_key": "eed02a0d-85a4-427b-851a-18dd8fd80d93",
+    "overwrite_quiet_hours": true
+  }
+}
+{{</ highlight >}}
+
+### Changes in Java package dependency
+
+With the release of Sensu Enterprise 3.0 the `sensu-enterprise` package
+dependency on the Java Virtual Machine will change from OpenJDK 1.7 to
+OpenJDK 1.8. This dependency will typically be satisfied
+automatically by your distribution's package management system,
+such as `yum` or `apt`.
+
+_NOTE: Users running Sensu Enterprise on RHEL/Centos 6 or similar
+distributions will need to install the [Extra Packages for
+Enterprise Linux (EPEL) repository][epel] to provide OpenJDK version
+1.8 when upgrading to Sensu Enterprise 3.0._
+
+The aim of this change is to help our customers stay up-to-date with
+their chosen Linux distributions and remain in compliance with security
+policies which may require packages like OpenJDK 1.7 to be removed
+once their end-of-support date is reached.
+
+Until recently [Redhat’s OpenJDK Life
+Cycle and Support Policy][rhel-openjdk-policy] listed June 2018 as
+the end-of-support date for OpenJDK 1.7. That date has since changed
+to June 2020.
```

Comment:
To keep this section short and avoid confusion, I'd recommend cutting this off after `...remain in compliance with security policies.` :)